### ISSUE TITLE
Image reading

### ISSUE THREAD

**Issue by JacksonCoder** [SouthEugeneRoboticsTeam/sobe#2]
So far, we have to manually specify images for the program. First, I think we need to modulate the code so the command line arguments can be separated from the main process of finding boxes in an image. Next, we also have to make a higher level program that can communicate with a video camera and pass the video to another function. There is also the issue of timing: the camera can supply images hundreds of times faster than we can process them, so we need to set an interval to process images (2 seconds or so). Multi-threading would help, so that we don't have to wait for an image to finish before starting to process the next one. Just to be clear, are we including the camera code in this repository, or is this just the processing part?

**Comment by andrewda** [comment]
This is everything that will run on the Jetson.

**Comment by JacksonCoder** [comment]
OK, good to know.

**Comment by andrewda** [comment]
This ticket has a lot of different sub-issues. I'm going to split this out into a couple different tickets.
